tags/cheatsheet/signatures.ytag

```diff
@@ -2,24 +2,23 @@ type: text
 
 ---
 
+*Regular Types*
 ```
 int -> I
-
 boolean -> Z
-
 void/no type -> V
-
 long -> J
-
 float -> F
-
 double -> D
-
 byte -> B
-
 char -> C
-
 short -> S
-
-array (type) -> [
 ```
+*Special Types*
+```
+object -> L<class>;
+array (type) -> [<type>
+```
+When specifying an object type it's important to add the fully qualified class name (which contains the package name) after the `L`, and adding a semicolon after that. For example, `Ljava/lang/String;` is a `String` object.
```

Comment:
You'd say `add a semicolon after that`, rather than `adding`, but I'd probably use `with a semicolon after that` here to avoid reusing words instead.